content/applications/services/helpdesk/overview/stages.rst

```diff
@@ -0,0 +1,113 @@
+======
+Stages
+======
+
+*Stages* are used to organize the *Helpdesk* pipeline and track the progress of tickets. Stages are
+customizable, and can be renamed to fit the needs of each team.
+
+Create or modify stages
+=======================
+
+.. important::
+   :ref:`Developer mode <developer-mode>` **must** be activated to access the stages menu. To
+   activate developer mode, go to :menuselection:`Settings app --> General Settings --> Developer
+   Tools`, and click :guilabel:`Activate the developer mode`.
+
+To view or modify *Helpdesk* stages, go to :menuselection:`Helpdesk app --> Configuration -->
+Stages`.
+
+The default list view on the :guilabel:`Stages` page displays the stages currently available in the
+*Helpdesk* app. They are listed in the order they appear in the pipeline.
+
+To change the order of the stages, click the :icon:`oi-draggable` :guilabel:`(draggable)` icon, to
+the left of the stage name, and drag it to the desired place on the list.
+
+.. image:: stages/stages-list-buttons.png
+   :alt: View of the stage list page emphasizing the buttons used to change the order the stages
+         appear in the list.
+
+.. tip::
+   Change the stage order on the Kanban view of a *Helpdesk* team's pipeline by dragging and
+   dropping individual columns.
+
+To create a new stage, click the :guilabel:`New` button at the top-left of the stage list. Doing so
+reveals a blank stage form.
+
+Choose a :guilabel:`Name` for the new stage, and add a description, if desired.
```

Comment:
There are more fields on the Stage form, and I'm not seeing any 'Description'

Can you double-check to ensure the accuracy of this statement during your revisions? Thanks!

![image](https://github.com/user-attachments/assets/d8eaf139-a09e-48b9-9c2e-4ba36376e78f)